.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
```

Comment:
Not sure if it'll run on tags if set up like this

Reply:
Yeah, there should be "tags:".

Reply:
Addressed in 988dfbc.